Week1/homework/js-exercises/2_errorDebugging.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+//console.log('I'm awesome'); cause of error is wrong use of second single quote. 
+
+//We can solve this problem with two ways:
+//First way:
+console.log("I'm awesome");
+
+//Second way:
+console.log("I\'m awesome");
```

Comment:
This works, but maybe you meant `console.log('I\'m awesome');` // Single quotes instead of double.